README.md

```diff
@@ -61,22 +61,28 @@ Group all DRM-related config. The currently available configs are:
 | `playback.getCurrentTime (deprecated)` |  | This method only exists for backward compatibility reasons. Prefer currentTime getter. |
 | `playback.getDuration (deprecated)` |  | This method only exists for backward compatibility reasons. Prefer duration getter. |
 | `playback.isPlaying (deprecated)` |  | This method only exists for backward compatibility reasons. Prefer playing getter. |
+| `playback.getPlaybackType (deprecated)` |  | This method only exists for backward compatibility reasons. Prefer mediaType getter. |
 
 | getter | description | response |
 |--------|-------------|----------|
+| `playback.config` | Returns the `options.html5TvsPlayback` value. | `{Object}` |
+| `playback.mediaType` | Returns if the media is `live` or `vod`. | `{String}` |
 | `playback.isReady` |  Indicates if the video is  ready to play, checking if [HTMLMediaElement.readyState](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState) is greater than or equal [HAVE_CURRENT_DATA](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState#value) value. | `{Boolean}` |
 | `playback.playing` | Indicates if the video is playing or not. | `{Boolean}` |
-| `playback.currentTime` |Returns the current position of the media is at the moment. | `{Number} - time in seconds` |
+| `playback.currentTime` | Returns the current position of the media is at the moment. | `{Number} - time in seconds` |
```

Comment:
I would phrase as `Returns the position the media is at the current moment`.

Reply:
Done here: 84c0ab7